### DISCUSSION TITLE
Please core team update the neovim-releases?

### DISCUSSION THREAD

**Discussion by tricktux** [neovim/neovim#31759] [General]
I understand is on best effort basis. But can somebody please update [neovim-releases](https://github.com/neovim/neovim-releases)?

It's of life of death to us mere mortals that have been marginalized to develop/maintain software on legacy distros. 😸 

Thank you for going the extra mile!

**Comment by tricktux** [comment]
@dundargoc please help a brother out with 10.4? 👍🏻 
Thanks for last time.

**Reply by SebasF1349** [reply]
I don't understand what are you asking for, the release section has the latest version.

**Reply by tricktux** [reply]
This: https://github.com/neovim/neovim-releases/releases

**Reply by clason** [reply]
I am sorry, but if it's life or death, you should compile from source yourself. It is really easy.
